packages/app/pages/Api/agent.tsx

```diff
@@ -35,9 +35,8 @@ export const Agent = (props) => {
   );
 };
 
-function concatArray(array: JSON[]) {
+export function concatArray(array: JSON[]) {
```

Comment:
Same comment as UP-94. I would move functions that are used by multiple files to a utility directory, or somewhere it is clear they don't belong to any one page. If everything requires from Agent, there is a lot of unnecessary dependency.

Reply:
will sync dev when up 94-is merged